### ISSUE TITLE
Javascript errors since Drupal 10 upgrade

### ISSUE THREAD

**Issue by bmunslow** [Gizra/drupal-starter#549]
Multiple Javascript errors are thrown in a clean drupal-starter install with Drupal 10.

```
drupal.js?v=10.1.2:64 Uncaught TypeError: $(...).once is not a function
    at Object.attach (header-menu.js?rzsw7a:16:50)
    at drupal.js?v=10.1.2:166:24
    at Array.forEach (<anonymous>)
    at Drupal.attachBehaviors (drupal.js?v=10.1.2:162:34)
    at big_pipe.js?v=10.1.2:118:10
    at big_pipe.js?v=10.1.2:138:3
```

It looks like several JS files in `server_theme` still have a dependency to the removed library `core/jquery.once`, they were not updated in the [Drupal 10 MR](https://github.com/Gizra/drupal-starter/pull/539):

```
web/themes/custom/server_theme/src/js/accordion.js
web/themes/custom/server_theme/src/js/header-menu.js
web/themes/custom/server_theme/src/js/search.js
web/themes/custom/server_theme/src/js/view-more.js
```

See:
- https://www.gizra.com/content/upgrading-drupal9-to-drupal10/
- https://www.drupal.org/node/3158256

**Comment by amitaibu** [comment]
@bmunslow Thanks

@mariano-dagostino ☝️

**Comment by amitaibu** [comment]
@bmunslow out of curiosity, are you using the repo on your projects?

**Comment by GizraNotifier** [comment]
The latest merged PR #550 just got deployed successfully to Pantheon [`qa`](https://gizra-drupal-starter-qa.pantheonsite.io) environment

**Comment by amitaibu** [comment]
> just got deployed successfully to Pantheon [qa](https://gizra-drupal-starter-qa.pantheonsite.io/) environment

@AronNovak The link to`qa` leads to 404

<img width="1041" alt="image" src="https://github.com/Gizra/drupal-starter/assets/125707/3e0ef077-47d0-4edc-a5c4-acacf45124a7">

**Comment by GizraNotifier** [comment]
The latest merged PR #564 just got deployed successfully to Pantheon [`qa`](https://qa-gizra-drupal-starter.pantheonsite.io) environment
